EIPS/eip-X.md

```diff
@@ -25,49 +26,60 @@ signature verification schemes.
 In order to delay unnecessary costs, Ethereum assets can continue using current cryptographic standards and only fall back
 to a quantum-secure scheme when there is reasonable risk of security failure due to quanutm computers.
 Therefore, this contract can server to protect one's funds on Ethereum by acting as a trigger that activates when 
 strong quantum supremacy has been achieved by solving the classically intractable puzzle.
 
+[protocols Link]: https://csrc.nist.gov/News/2022/pqc-candidates-to-be-standardized-and-round-4
+
 
 ## Specification
 
 ### Puzzle
 
-The puzzle that this contract will generate is one of [order-finding](https://en.wikipedia.org/wiki/Shor%27s_algorithm#Quantum_order-finding_subroutine),
+The puzzle that this contract generates is one of order-finding,
 where given a positive integer _n_ and an integer _a_ coprime to _n_, the objective is to find the smallest positive integer
 _k_ such that _a_ ^ _k_ = 1 (mod _n_).
 
+[order-finding Link]: https://en.wikipedia.org/wiki/Shor%27s_algorithm#Quantum_order-finding_subroutine
+
 
 ### Requirements
 
 - This contract SHALL generate 1 integer, the modulus, of exactly 784 random bits. 
   It SHALL then generate another integer, the base, of <= 784 bits and reduce it modulo the first generated integer.
 - If the base is equal to 1 or -1 mod _n_ or is coprime to the modulus, it MUST be thrown out and another base MUST be generated.
 - This contract MUST accept funds from any account without restriction.
-- This contract MUST allow someone to provide the [multiplicative order](https://en.wikipedia.org/wiki/Multiplicative_order) of the base with the modulus.
+- This contract MUST allow someone to provide the multiplicative order of the base with the modulus.
   If it is the correct solution, then this contract MUST send all of its funds to the solver and mark a flag to indicate that this contract has been solved.
 
+[multiplicative order Link]: https://en.wikipedia.org/wiki/Multiplicative_order
+
 ### Deployment method
 
 - The contract MUST be deployed as a [Singleton][ERC-2470]
 - After deploying the contract with parameters of 1 lock having a 784-bit modulus, the contract's `triggerLockAccumulation()` method SHALL be called repeatedly until all bits have been generated.
 
 ### Providing solutions
 
-- Providing solutions MUST follow a [commit-reveal](https://medium.com/swlh/exploring-commit-reveal-schemes-on-ethereum-c4ff5a777db8) scheme to prevent [front running](https://solidity-by-example.org/hacks/front-running/.).
+- Providing solutions MUST follow a commit-reveal scheme to prevent front running.
 - This scheme MUST require one day between commit and reveal.
 
+[commit-reveal Link]: https://medium.com/swlh/exploring-commit-reveal-schemes-on-ethereum-c4ff5a777db8
+[front running Link]: https://solidity-by-example.org/hacks/front-running/
+
 ### Bounty funds
 
 - Funds of at least 50,000 gas SHALL be sent to the contract as a bounty.
```

Comment:
Not super clear on this point. Gas and balance are different units. You should specify the minimum bounty size in terms of balance (amount of wei, gwei, etc). The minimum gas price on mainnet is 7 wei post 1559, but you should assume mainnet is closer to 20 gwei. So 20 gwei * 50,000 would roughly the minimum bounty you would want to provide.

Reply:
The issue is if the price increases, the bounty should be increased as well. The bounty should cover 50,000 gas at all times.